.github/workflows/LinuxRelease.yml

```diff
@@ -281,7 +283,7 @@ jobs:
       shell: bash
       run: |
         mkdir -p build/release
-        (cd build/release && cmake -DSTATIC_LIBCPP=1 -DBUILD_ICU_EXTENSION=1 -DBUILD_PARQUET_EXTENSION=1 -DBUILD_FTS_EXTENSION=1 -DBUILD_JSON_EXTENSION=1 -DBUILD_EXCEL_EXTENSION=1 -DFORCE_32_BIT=1 -DCMAKE_BUILD_TYPE=Release ../.. && cmake --build .)
+        (cd build/release && cmake -DSTATIC_LIBCPP=1 -DBUILD_ICU_EXTENSION=1 -DBUILD_PARQUET_EXTENSION=1 -DBUILD_FTS_EXTENSION=1 -DBUILD_JSON_EXTENSION=1 -DBUILD_EXCEL_EXTENSION=1 -DBUILD_SQLSMITH=ON -DFORCE_32_BIT=1 -DCMAKE_BUILD_TYPE=Release ../.. && cmake --build .)
```

Comment:
Unsure, but I think this will make for shipping SQLsmith with releases, and probably that can be avoided (both here and on the linux-release-aarch64 job). Probably having it enable only for Main.yml should be enough.

Reply:
Agreed - adding it to the extension build might also cause it to be shipped by accident. @PedroTadim could you perhaps remove all instances of `BUILD_SQLSMITH_EXTENSION` except in the `Linux (64 Bit)` run? That should be sufficient to catch compilation failures early.

Reply:
I was not aware that the script was used to build production binaries. Time to fix